runtime/drivers/clickhouse/clickhouse.go

```diff
@@ -507,39 +507,70 @@ func (c *Connection) lastUsedOn() time.Time {
 }
 
 // Periodically collects stats about the database and emit them as activity events.
-func (c *Connection) periodicallyEmitStats(d time.Duration) {
+func (c *Connection) periodicallyEmitStats(sensitive, regular time.Duration) {
```

Comment:
Now that there are multiple, maybe just drop the params here and hard-code in the loop